src/WebJobs.Script/Diagnostics/MetricEventNames.cs

```diff
@@ -9,6 +9,8 @@ public static class MetricEventNames
         public const string ApplicationStartLatency = "host.application.start";
         public const string ApplicationInsightsEnabled = "host.applicationinsights.enabled";
         public const string ApplicationInsightsDisabled = "host.applicationinsights.disabled";
+        public const string OpenTelemetryAzMonEnabled = "host.opentelemetry.azmon.enabled";
```

Comment:
What do you think of using `otel`? I am good either way, just want to make sure we discuss our options. Especially considering you are already abbreviating `azmon` and `otlp`.

```suggestion
        public const string OpenTelemetryAzMonEnabled = "host.otel.azmon.enabled";
```